tools/xcodeproj_shims/build-wrapper.sh

```diff
@@ -22,6 +22,9 @@ fi
 if [ -n "${TARGET_DEVICE_IDENTIFIER:-}" ] && [ "$PLATFORM_NAME" = "iphoneos" ]; then
     echo "Builds with --ios_multi_cpus=arm64 since the target is an iOS device."
     BAZEL_BUILD_OPTIONS+=("--ios_multi_cpus=arm64")
+elif [ $FORCE_X86_SIM -gt 0 ] && [ "$PLATFORM_NAME" = "iphonesimulator" ] && [[ $(sysctl -n machdep.cpu.brand_string) =~ "Apple" ]]; then
```

Comment:
We probably do not need the sysctl check.
It wont hurt setting it to x86_64 even on Intel machines.